src/pages/home/report/comment/TextCommentFragment.tsx

```diff
@@ -72,10 +65,13 @@ function TextCommentFragment(props) {
         );
     }
 
+    const propsStyle = Array.isArray(props.style) ? props.style : [props.style];
+
     const containsOnlyEmojis = EmojiUtils.containsOnlyEmojis(text);
+    const message = isEmpty(iouMessage) ? text : iouMessage;
 
     return (
-        <Text style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style]}>
+        <Text style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...propsStyle]}>
```

Comment:
RN styles handles falsy values, as well as nested objects:
```suggestion
        <Text style={[containsOnlyEmojis && styles.onlyEmojisText, styles.ltr, style]}>
```